projects/ks89/angular-modal-gallery/src/lib/components/previews/previews.component.ts

```diff
@@ -170,28 +170,12 @@ export class PreviewsComponent extends AccessibleComponent implements OnInit, On
    * In particular, it's called when any data-bound property of a directive changes!!!
    */
   ngOnChanges(changes: SimpleChanges): void {
```

Comment:
In order for the previews calculation to be more consistent and simple, I simplified this part so that the previews are calculated, with any change, by the same process.